File: Foundation/Scratch/Getting Started/c++/l002prime.cpp
// bool isPrime(int n)
// {
//     for(int i = 2; i*i < n; i++){
//         if(n%i){
//           return false;
//           break;}
//     }
//     return true;
// }

// void primenum(){
//     int t,n ;
//     cin>> t;
//     for(int i =1; i<=t; i++)
//     {
//         cin>>n;
//         if(isPrime(n)){
//             cout<<"prime";
//         }
//         else{
//             cout<<"not prime";
//         }
//     }
// }

#include <iostream>
using namespace std;
bool isPrime(int n)
{
    for(int i = 2; i*i <= n; i++){
        if(n%i==0){
          return false;
        }
    }
    return true;
}

void primenum(){
    int t,n ;
    cin>> t;
    for(int i =1; i<=t; i++)
    {
        cin>>n;
        if(isPrime(n)){
            cout<<"prime"<<endl;
        }
        else{
            cout<<"not prime"<<endl;
        }
    }
}
int main(int argc, char **argv){
    primenum();

    //write your code here
    
}

File: Foundation/Scratch/Getting Started/java/l002Prime.java

import java.util.*;

public class Main {
    public static Scanner scn = new Scanner(System.in);
    public static boolean isPrime(int n)
{
    for(int i = 2; i*i <= n; i++){
        if(n%i==0){
          return false;
        }
    }
    return true;
}

public static void primenum(){
    int t= scn.nextInt() ;
      for(int i =1; i<=t; i++)
    {
       int n= scn.nextInt(); 
        if(isPrime(n)){
            System.out.println("prime");
        }
        else{
            System.out.println("not prime");
        }
    }
}

    public static void main(String[] args) {
        
primenum();
        // write ur code here

    }
}
